### PR TITLE
MultiQC v1.13a0 

### PR DESCRIPTION
**Pull request by apeltzer**
(dev version) with bugfixes for mirtop… … and Dupradar

Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/multiqc/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 package:
   name: multiqc
   version: {{ version }}
 
 source:
-  url: https://github.com/ewels/MultiQC/archive/v{{ version }}.tar.gz
-  sha256: 0dc332ef2e79f0f950d745104c948d8b1c07f3b43648d7331d6919c56cd42834
+  #url: https://github.com/ewels/MultiQC/archive/v{{ version }}.tar.gz
+  #sha256: 0dc332ef2e79f0f950d745104c948d8b1c07f3b43648d7331d6919c56cd42834
+  git_url: https://github.com/ewels/MultiQC.git
```
[the remark this answers is not in the document]
Comment (apeltzer) [reply recipes/multiqc/meta.yaml:10]: Wasn't sure if git_url can even have it, so wanted to let it try first and add if not ;-)

[the remark this answers is not in the document]
Comment (ewels) [reply recipes/multiqc/meta.yaml]: I think that the GitHub zip file / tarball should work: https://github.com/ewels/MultiQC/archive/refs/heads/master.zip

**Review by ewels — APPROVED** [review]
Nice 🚀

**Comment by apeltzer** [comment]
Any idea @ewels ? Something about module not being callable - just added rich-click... which is required now?

**Comment by apeltzer** [comment]
@bioconda/python -- any idea why the module import does not work?

**Comment by apeltzer** [comment]
@BiocondaBot please merge

**Comment by bgruening** [comment]
`dev` version makes me nervouis. Please note that people will get this version by default if they do not specify a version. Those version should be stable imho.

**Comment by apeltzer** [comment]
This dev release is stable + two bugfixes and @ewels and I discussed to have a release as the bugs were really bugging a couple of people ;-) Will definitely release a 1.13 full release once there is a final release asap.

**Comment by ewels** [comment]
Or... ok maybe not so stable. Looks like this is failing with something odd-sounding from the Conda build. Follow-up PR: https://github.com/bioconda/bioconda-recipes/pull/35553

**Pull request merged**